Moli/API/UserAPI/MySubPath.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import UIKit
+
+extension UserAPI {
+    static func uploadBannerImage(image: UIImage) async throws {
+        guard let data = image.jpegData(compressionQuality: 1.0) else {
+            print("Unable to get image data in uploadBannerImage")
+            return
+        }
+        let formdataRequest = MultipartFormDataRequest(urlComponents: userURL(subPath: "/my/profile/banner_image"))
+        formdataRequest.addDataField(
+            named: "banner_image",
+            filename: "wallaspace.jpg",
```

Comment:
:art: Kanske ska heta något mer professionellt? xD
```suggestion
            filename: "banner_image.jpg",
```

Reply:
Denna skickas till backenden och används i övrigt inte alls. Den behövs bara för att filename måste skickas med för att kunna ladda upp filen till backend, även om filename inte används. Men kan ändra ändå :)